packages/gatsby-plugin-utils/src/polyfill-remote-file/jobs/dispatchers.ts

```diff
@@ -96,7 +98,7 @@ export function dispatchLocalImageServiceJob(
       outputDir: path.join(
         global.__GATSBY?.root || process.cwd(),
         ...publicUrl.filter(Boolean),
-        generateImageArgs({ width, height, format, quality })
+        generateImageArgs({ width, height, format, quality, aspectRatio })
```

Comment:
We don't need it here, aspectratio should be calculated already
```suggestion
        generateImageArgs({ width, height, format, quality })
```

Reply:
Fixed!